plugins/rss/init.js

```diff
@@ -179,7 +179,12 @@ theWebUI.config = function()
 	this.rssLabels = {};
 	this.rssItems = {};
 	this.rssGroups = {};
-	this.updateRSSTimer = null;
+	this.actRSSLbl = null;
+	this.updateRSSTimer = 0;
+	this.rsstimerUpdateIntervalId = 0;
+	this.rsstimerUpdateIntervalId = 0;
+	this.updateRSSInterval = 5*60*1000;
```

Comment:
Duplicate variable.

Reply:
`this.updateRSSInterval` is duplicated again two lines bellow.

Reply:
Thanks! This has been a really strange merge conflict diff :smile: